ush/templates/FV3LAM_wflow.xml

```diff
@@ -31,15 +31,16 @@ Workflow task names.
 <!ENTITY MAKE_SFC_CLIMO_TN "{{ make_sfc_climo_tn }}">
 <!ENTITY GET_EXTRN_ICS_TN  "{{ get_extrn_ics_tn }}">
 <!ENTITY GET_EXTRN_LBCS_TN "{{ get_extrn_lbcs_tn }}">
+<!ENTITY GET_EXTRN_LBCS_LONG_TN "{{ get_extrn_lbcs_long_tn }}">
 <!ENTITY MAKE_ICS_TN       "{{ make_ics_tn }}">
 <!ENTITY MAKE_LBCS_TN      "{{ make_lbcs_tn }}">
 <!ENTITY RUN_FCST_TN       "{{ run_fcst_tn }}">
 <!ENTITY RUN_POST_TN       "{{ run_post_tn }}">
+<!ENTITY ANAL_GSI_TN       "{{ anal_gsi }}">
+<!ENTITY PREP_COLDSTART_TN "{{ prep_coldstart }}">
+<!ENTITY PREP_WARMSTART_TN "{{ prep_warmstart }}">
 
-
-<!ENTITY ANAL_GSI_INPUT_TN    "{{ anal_gsi_input }}">
-<!ENTITY ANAL_GSI_RESTART_TN    "{{ anal_gsi_restart }}">
-
+<!ENTITY RUN_PREPSTART_TN "run_prepstart">
```

Comment:
Please add white space to align with values below.

Reply:
Done